core/src/main/java/io/undertow/util/Certificates.java

```diff
@@ -40,7 +37,18 @@ public static String toPem(final X509Certificate certificate) throws Certificate
         builder.append(END_CERT);
         return builder.toString();
     }
-
+/*
+    @Deprecated
+    public static String toPem(final javax.security.cert.X509Certificate certificate) throws javax.security.cert.CertificateEncodingException {
+        final StringBuilder builder = new StringBuilder();
+        builder.append(BEGIN_CERT);
+        builder.append('\n');
+        builder.append(FlexBase64.encodeString(certificate.getEncoded(), true));
+        builder.append('\n');
+        builder.append(END_CERT);
+        return builder.toString();
+    }
+*/
```

Comment:
commented out code block?

Reply:
Was tracing internal usage.  Forgot to re-enable in case anyone else was using this public API.  Fixed.

Reply:
I can't find where this is fixed?

Reply:
Ok, now I can find where it is fixed! The < and > navigation of the PR review screen did not show those commits :-)